### DISCUSSION TITLE
Timestamp Overlay

### DISCUSSION THREAD

**Discussion by aliciahps** [gopro/labs#417] [General]
Is there a way to use the timestamp overlay on a GP 11 mini? I've tried using this one but I haven't been able to get it to work. Any tips?

https://[gopro.github.io/labs/control/overlays/](https://gopro.github.io/labs/control/overlays/)

**Comment by dnewman-gpsw** [comment]
It works, but only at 4Kp30 or lower res and fps.

**Comment by aliciahps** [comment]
> It works, but only at 4Kp30 or lower res and fps.

I've changed the settings and tried scanning the QR it reads and says the overlay is active but does not show up when the recording is over or during the preview before recording.

**Comment by dnewman-gpsw** [comment]
It seems the Mini no longer supports overlays on 4k30, but found it works on 1080p60 (maybe other resolutions work.) There are complex reasons that may have been changed in the base firmware, impacted Labs.  Overlays is one of the features not fully supported on GP2, as it is a hack developed for GP1, so it only works in video modes configured like GP1 (high level, as the details can't be shared.) 

You can try older firmware, as this might be recent change.  But current firmware is generally the best option.

**Comment by dnewman-gpsw** [comment]
Good news, I was wrong. I did my testing with 10-bit mode.  4Kp30 16x9 8-bit does work with overlays.  Only 8-bit modes will work (and 4k30 16x9 or less.)

**Comment by dnewman-gpsw** [comment]
As this is not a bug, I moved to discussions.

**Comment by StanMango** [comment]
Hi. I got a hero 11 in order to do timelapse for an experiment. I need to timestamp the video. I have discovered it is not currently supported. Is it likely to be supported in future? Is there somewhere I should be requesting its support? Or is there a way to extract the timestamp of each frame as metadata?

**Reply by StanMango** [reply]
Another option is I get a hero 9. Can someone confirm overlay (BURN) is supported in timelapse?

**Reply by dnewman-gpsw** [reply]
I can confirm it works fine on HERO9.  HERO10/11 is not going to see this fixed, as internally the GP2 architecture is different enough to make this Labs hack difficult.

**Reply by StanMango** [reply]
Out of curiosity, if I understand correctly: the 'function' BURN is written by gopro labs and is in the firmware that we upload from gopro labs. BURN then interacts with the GP2 architecture to achieve the overlay. But differences in GP2 from its predecessor mean BURN only works for some video resolutions and speeds?

**Reply by dnewman-gpsw** [reply]
Yes.  GP2 adds 10-bit and other more complex pixels formats that aren't compatibility with overlay engine build for GP1.

**Reply by dnewman-gpsw** [reply]
I forgot to add, "is there a way to extract the timestamp of each frame as metadata" , yes this is the normal way to do it.

**Comment by zeeedi** [comment]
Hi. Does it mean that the burn in overlay (e.g. time/date) is not working for the Hero 11 black?

**Reply by dnewman-gpsw** [reply]
It works on HERO11 in normal video mode 4Kp30 or fewer pixels/s.  However is does not work in the timelapse mode.

**Reply by rmarsh3309** [reply]
I have not been able to get the "Burn-in" A.K.A. Time-Date overlay to function on my HERO 11 Black.  I've tried every possible resolution.  It appears that it will only work in 8-Bit video?  There's no way to turn off 10-Bit video on my HERO 11.  So..... is anybody getting Time-Date overlay working on a Hero 11?

**Reply by dnewman-gpsw** [reply]
Under Protune turn 10-bit off

**Reply by rmarsh3309** [reply]
David, Have you been able to do that with a Hero 11?  I see the setting, but no matter which resolution and frame rate, it can't be changed.

**Reply by dnewman-gpsw** [reply]
Weird, HERO11 has no 10-bit only modes.  Factory reset the camera, and try again.  It should even default to 10-bit off.

**Reply by rmarsh3309** [reply]
David, thanks so much man!  The Factory reset did the trick.
